### ISSUE TITLE
Question: Command history in qalc (CLI)

### ISSUE THREAD

**Issue by abrac** [Qalculate/libqalculate#330]
How do you get the command history in `qalc`? I know how to do it in qalculate-gtk.

It would be nice to have a `history()` comand, for example. If I wanted the last 20 commands, I would type `history(20)`.

**Comment by hanna-kn** [comment]
I'm not sure I understand the question/request. I would just scroll up or use Up/Down keys in "interactive" mode. I guess something similar to the latter would be useful for non-interactive mode (if the history of the shell is full of unrelated commands). This could be implemented by adding expressions provided on the shell command line to the history (which I'm hesitant about) and/or a separate history command line switch (which would list the last 20 or so expressions provided on the command line).

**Comment by abrac** [comment]
I just wanted it for record keeping purposes. 

For example, let's say I used qalc to do a set of calculations for a particular project. I just wanted an easy way to copy and paste that set of calculations, so that I can keep a record of them.

However, I have found a solution: I found that the command history is saved in `~/.config/qalculate/qalc.history`. It was easy for me to copy and paste the set of commands from that file. 

Thank you for your help. 😊😊 I will close this issue.
